src/less/menu/base/menu.less

```diff
@@ -82,7 +83,7 @@ div.menu__item[role^="menuitem"][aria-checked="true"] svg.icon {
 a.fake-menu__item span,
 button.fake-menu__item span,
 div.menu__item[role^="menuitem"] span {
-    flex: 1 0 auto;
+    flex: 0 0 auto;
```

Comment:
Curious as to what this change was for?

Reply:
I don't know why this worked, but it totally fixed the issue.

Reply:
Ok, just double check for any regressions in non RTL mode

Reply:
✅